api/src/main/java/com/tersesystems/echopraxia/spi/DefaultToStringFormatter.java

```diff
@@ -28,10 +28,26 @@ public String formatField(@NotNull Field field) {
       assert fieldVisitor != null;
       field = fieldVisitor.visit(field);
     }
-    formatValue(builder, field.value(), attributes);
+
+    Attributes valueAttributes = collectValueAttributes(field.attributes(), field.value().attributes());
+    formatValue(builder, field.value(), valueAttributes);
     return builder.toString();
   }
 
+  // if the field has elided, tostringvalue, abbreviateafter, or ascardinal,
+  // it should apply to the value instead.
+  Attributes collectValueAttributes(Attributes fa, Attributes valueAttributes) {
```

Comment:
Add a fallback for fields that have attributes set on them that really only apply to the value.